tests/queries/0_stateless/03594_text_index_size_in_data_skipping_indices.reference

```diff
@@ -0,0 +1,19 @@
+Row 1:
+──────
+database:                default
+table:                   tab
+name:                    text_idx
+type:                    text
+type_full:               text(tokenizer = 'ngram', ngram_size = 3)
+expr:                    str
+granularity:             1
+data_compressed_bytes:   133
```

Comment:
FastTest failed because the byte sizes are different in CI.
This is expected due to platform differences.

I think it will be more robust to do

```sql
SELECT data_compressed_bytes > 50, data_uncompressed_bytes > 50 FROM system.data_skipping_indices
```

instead of

```sql
SELECT * FROM system.data_skipping_indices
```

Reply:
... due to platform differences _and_ [randomized settings](https://github.com/ClickHouse/ClickHouse/blob/master/tests/clickhouse-test#L1020-L1151) in CI.